Plotly.Blazor/PlotlyChart.razor

```diff
@@ -37,6 +37,12 @@
     /// </summary>
```

Comment:
A reminder these same changes would need to be made in the .Generator project as well